packages/neuron-ui/src/components/PageContainer/pageContainer.module.scss

```diff
@@ -5,7 +5,6 @@
   position: relative;
   display: flex;
   flex-direction: column;
-  min-height: 100vh;
   height: 100vh;
 
```

Comment:
`min-height: 100vh;`  should not be removed.

Reply:
The next line defined height as `100vh`. Why should we define `min-height`?

Reply:
If remove  `min-height: 100vh; `。some pages's header cann't be stickied (eg: transation history page)

<img width="1328" alt="image" src="https://github.com/nervosnetwork/neuron/assets/11756869/9b3dfc2c-e186-48a1-83f9-3a769d046c2e">

Reply:
It seems no problem with the transaction history page.

https://github.com/nervosnetwork/neuron/assets/12881040/510684b2-40af-4908-8ed2-a745d39b72ac